### PR TITLE
test native fixes for roundtripping placement when no geometry is present

### PR DESCRIPTION
**Pull request by MichaelBelousov**
- hoist `deepEqualWithFpTolerance` to the backend tests so it can be used there and in transformer tests
- minor jsdoc fix in iModelDb I noticed
- tests for this [native pr](https://dev.azure.com/bentleycs/iModelTechnologies/_git/imodel02/pullrequest/248644)

### CODE REVIEW

File: test-apps/imodel-transformer/src/Transformer.ts
Comment (MichaelBelousov) [inline test-apps/imodel-transformer/src/Transformer.ts:69]: this PR was based off the isolated elements PR to the transformer test-app, will rebase out those changes

**Review by pmconne — APPROVED** [review]

**Comment by MichaelBelousov** [comment]
will merge after native PR merges

**Pull request merged**
